BibBuddy/app/src/androidTest/java/de/bibbuddy/NotesFragmentTest.java

```diff
@@ -1,64 +1,93 @@
-/*
 package de.bibbuddy;
 
 import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.Espresso.openActionBarOverflowOrOptionsMenu;
 import static androidx.test.espresso.action.ViewActions.click;
 import static androidx.test.espresso.action.ViewActions.swipeLeft;
 import static androidx.test.espresso.assertion.ViewAssertions.matches;
-import static androidx.test.espresso.matcher.ViewMatchers.hasDescendant;
 import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
 import static androidx.test.espresso.matcher.ViewMatchers.withText;
 
-
 import android.content.res.Resources;
 import android.graphics.drawable.Drawable;
 import android.view.View;
 import android.widget.ImageView;
 import androidx.recyclerview.widget.RecyclerView;
 import androidx.test.core.app.ActivityScenario;
-import androidx.test.espresso.action.ViewActions;
-import androidx.test.espresso.contrib.RecyclerViewActions;
 import androidx.test.espresso.matcher.ViewMatchers;
 import java.util.Objects;
 import org.hamcrest.Description;
 import org.hamcrest.TypeSafeMatcher;
 import org.junit.Before;
 import org.junit.Test;
 
+/**
+ * NotesFragmentTest is responsible for the UI-test of the NotesFragment-class.
+ *
+ * @author Sabrina.
+ */
 public class NotesFragmentTest {
 
   static NotesFragment nF;
   private static View itemView;
-  private final String exampleText = "text";
-  private String modDate;
-  private String name;
-  private int idText;
+  private int itemViewId;
 
   /**
-   * This method initiates the basic NotesFragment setup before every test-case.
+   * Initiate basic NotesFragment' setup before every test-case.
    */
-/*
   @Before
   public void init() {
     ActivityScenario scenario = ActivityScenario.launch(MainActivity.class);
     onView(withId(R.id.navigation_notes)).perform(click());
-    onView(withId(R.id.fragment_notes)).check(matches(isDisplayed()));
-    ActivityScenario activityScenario = scenario.onActivity(
+    scenario.onActivity(
         (ActivityScenario.ActivityAction<MainActivity>) activity -> {
           nF = (NotesFragment) activity.getSupportFragmentManager().getFragments().get(0);
-          NotesFragment.noteList.clear();
-          NoteModel noteModel = new NoteModel(nF.getContext());
-          noteModel.createNote(exampleText, 0, exampleText,"");
-          NotesFragment.noteList = noteModel.getNoteList();
+
+          if (nF.noteList.size() != 0) {
+            RecyclerView recyclerView = nF.getView().findViewById(R.id.note_list_recycler_view);
+            itemView = recyclerView.getChildAt(0);
+            itemViewId = itemView.getId();
+          }
```

Comment:
Suggest adding a new line after line 51, because before the if()-block there is a new line.

Reply:
Done.